NepaliCalendar/NepaliCalendar/DateConverter/DateConverter.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DateConverter.swift
+//  NepaliCalendar
+//
+//  Created by alok subedi on 01/05/2021.
+//
+
+public final class DateConverter {
+    
+    public static func bSToAD(date: NCDate) throws -> (day: Int, month: Int, year: Int) {
+        if !validateDate(date: date) {
+            throw NSError()
+        }
+        
+        return (0,0,0)
+    }
+    
+    private static func validateDate(date: NCDate) -> Bool {
```

Comment:
Func name `validateDate(date:)` defies swift's standard for naming. Use either `validate(date:)` or `validateDate(_:)`